pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -14,7 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package unversioned_test
```

Comment:
Ah gotcha, very sorry, clearly I did not read the above very carefully at all :)

btw, I believe your suggestion of "unversionedTest" will not compile, since there are other files in package "unversioned" in the directory.

But ordinarily, I agree that we wouldn't use "_" in a package name.